dask/dataframe/tests/test_dataframe.py

```diff
@@ -2077,7 +2079,7 @@ def test_repartition_on_pandas_dataframe():
 def test_repartition_npartitions(use_index, n, k, dtype, transform):
     df = pd.DataFrame(
         {"x": [1, 2, 3, 4, 5, 6] * 10, "y": list("abdabd") * 10},
-        index=pd.Series([1, 2, 3, 4, 5, 6] * 10, dtype=dtype),
+        index=pd.Series([10, 20, 30, 40, 50, 60] * 10, dtype=dtype),
```

Comment:
This broke tests on aarch64, ppc64le, s390x again, cf #4561

Reply:
Thanks for the report @QuLogic, is the problem the same? It seems pretty unsatisfying to have to change the specific numbers here in order to support those architectures.

Reply:
Thanks for pointing this out @QuLogic - I didn't realize that this was changed from `[10, 20, 30, 40, 50, 60]` to `[1, 2, 3, 4, 5, 6]` in the past. Unfortunately, using `[1, 2, 3, 4, 5, 6]` no longer works when `from_pandas` returns the *correct* number of partitions, so I cannot simply revert this change.

Can you share what parameters/assertions are failing? Does it help if we remove the `assert all(map(len, parts))` (since it's technically *okay* to have empty partitions if we are increasing the number of partitions)?

Reply:
The same ones as before: `test_repartition_npartitions[<lambda>0-float-5-1-True]` and `test_repartition_npartitions[<lambda>1-float-5-1-True]`. See [build.log on the other arches here](https://koji.fedoraproject.org/koji/taskinfo?taskID=90774706)

Reply:
Ping? This is still broken in the latest version. Should I open a new issue?

Reply:
Thanks for the ping @QuLogic. A new issue would be good. Do you have a traceback?

Reply:
Should be the same as before #9582.